swim/memberlist_test.go

```diff
@@ -66,6 +66,34 @@ func (s *MemberlistTestSuite) SetupTest() {
 	}
 }
 
+func (s *MemberlistTestSuite) TestAddJoinList() {
+
+	joinList := s.changes
+
+	// We add all the members of this node to the joinList because we want
+	// the joinList to contain the change for the local member.
+	mems := s.node.disseminator.FullSync()
+	for _, mem := range mems {
+		joinList = append(joinList, mem)
+	}
+
+	// We make sure that the joinList contains a change for the local member.
+	var localMember *Change
+	for _, mem := range joinList {
+		if mem.Address == s.node.Address() {
+			localMember = &mem
+		}
+	}
+	s.NotNil(localMember, "expected joinList to contain a change for the local member")
```

Comment:
Every time I see `NotNil`, I read it as `NotNils`. And I think: "What has Nils done _this_ time?!" :smirk: